packages/tupaia-web/src/AppProviders.tsx

```diff
@@ -15,6 +15,7 @@ const queryClient = new QueryClient({
       staleTime: 1000 * 60 * 5, // 5 minutes
       refetchOnWindowFocus: false,
       retry: 1,
+      keepPreviousData: true,
```

Comment:
This means that once data is loaded for a query it keeps that data until the next query is done and stops any flickering that would occur.